bin/dummy

```diff
@@ -0,0 +1,78 @@
+#!/usr/bin/env rails runner
+organization = Organization.create!(name: 'Dummy organization-213')
+student_role = organization.roles.create!(group: Person::STUDENT, title: 'student-213')
+mentor_role = organization.roles.create!(group: Person::MENTOR, title: 'mentor-213')
+
+students =[
+  { first_name: 'Bram', last_name: 'Moolenaar', gender: 'male' },
+  { first_name: 'Edsger', last_name: 'Dijkstra', gender: 'male' },
+  { first_name: 'Andrew', last_name: 'Tanenbaum', gender: 'male' }
+]
+phone = nil
+students = students.map do |student_hash|
+  loop do
+    phone = "06#{rand(10 ** 8).to_s.rjust(8,'0')}"
+    break if Person.find_by_mobile_phone(phone).nil?
+  end
+  Person.create!(first_name: student_hash[:first_name],
+                  last_name: student_hash[:last_name],
+                  gender: student_hash[:gender],
+                  mobile_phone: phone, role: student_role)
+end
+
+loop do
+  phone = "06#{rand(10 ** 8).to_s.rjust(8,'0')}"
+  break if Person.find_by_mobile_phone(phone).nil?
+end
+
+mentor = Person.create!(first_name: 'Guido',
+                last_name: 'van Rossum',
+                gender: 'male',
+                email: 'janssens_hermanus81@gmail.com',
+                mobile_phone: phone, role: mentor_role)
+
+
+protocol = Protocol.find_by_name('mentoren dagboek')
+
+responseobj = nil
+students[0..1].each do |student|
+  prot_sub = ProtocolSubscription.create!(
+    protocol: protocol,
+    person: mentor,
+    filling_out_for: student,
+    state: ProtocolSubscription::ACTIVE_STATE,
+    start_date: Time.zone.now.beginning_of_week
+  )
+  responseobj = prot_sub.responses.first
+  responseobj.update_attributes!(
+    open_from: 1.minute.ago,
+    invited_state: Response::SENT_STATE)
+  token = responseobj.initialize_invitation_token!
+end
+tok = "#{responseobj.protocol_subscription.person.external_identifier}#{responseobj.invitation_token.token_plain}"
+`bin/dummy_mentor #{tok}`
```

Comment:
kunnen we hier iets van `require File.expand_path(..., __dir__)` gebruiken om het pad van dummy_mentor te bepalen? Of gaat dit ook goed wanneer je vanuit een andere directory dan rails_root dit script aanroept?

Reply:
`file_name = Rails.root.join('bin','dummy_mentor').to_s` heb ik ervan gemaakt